src/lib/mixer_module/output_functions.yaml

```diff
@@ -36,6 +36,8 @@ functions:
 
     Gripper: 430
 
+    Landing_Gear_Wheel: 440
```

Comment:
@bkueng is this number fine? Ideally it would be grouped with Landing_Gear, Parachute as "external flight control actuators" (vs payload). But changing already assigned functions is probably not possible. wdyt?

Reply:
Yes, fine with me, I'm not sure if we can just change the display ordering.

Reply:
I wouldn't change the display order, then it gets hard to see which values are still available.